src/components/evaluation/charts/PieChart.js

```diff
@@ -72,14 +72,42 @@ const PieChart = ({ isSolutionShown, data }) => (
 PieChart.propTypes = propTypes
 PieChart.defaultProps = defaultProps
 
+const indexToLetter = index =>
```

Comment:
please extract these functions into a separate file (lib/charts.js). also the functions in the bar chart.. they could be reusable for something else, so we should extract them.

Reply:
Done :)